Makefile

```diff
@@ -21,17 +21,29 @@ DATE    ?= $(shell git log -1 --format="%ct" | xargs -I{} date -d @{} +'%Y-%m-%d
 CNINFRA := go.ligato.io/cn-infra/v2/agent
 LDFLAGS = -X $(CNINFRA).BuildVersion=$(VERSION) -X $(CNINFRA).CommitHash=$(COMMIT) -X $(CNINFRA).BuildDate=$(DATE)
 
+RELEASE_TAG ?= $(shell git describe --always --tags --dirty --exact-match 2>/dev/null)
+
+TAG_FORMAT="^v([0-9]+\.){2}[0-9]+.*"
+RELEASE_TAG_CHECKED = $(shell echo $(RELEASE_TAG) | grep -v "\-dirty" | grep -E ${TAG_FORMAT})
+ifneq ($(RELEASE_TAG_CHECKED),)
+RELEASE_VERSION_FULL = $(shell echo $(RELEASE_TAG_CHECKED) | cut -c 2-)
+RELEASE_VERSION_MAJOR_MINOR = $(shell echo $(RELEASE_VERSION_FULL) | cut -d '.' -f 1-2)
+endif
+
 ifeq ($(VPP_VERSION),)
 VPP_VERSION="21.06"
 endif
+ifeq ($(DEV_VERSION),) # for tagging in-development images
+DEV_VERSION="21.06.0"
```

Comment:
This should be `21.06`. The `21.06.0` tag is a unique tag which is built and pushed only once on the actual git tag `v21.06.0`. Unlike the `21.06` tag, which is a rolling tag that gets updated to the latest patch release of `v21.06.XX` version.